plugins/inputs/mesos/mesos.go

```diff
@@ -450,10 +453,9 @@ func (m *Mesos) gatherSlaveTaskMetrics(address string, defaultPort string, acc t
 			return err
 		}
 
+		jf.Fields["framework_id"] = task.FrameworkID
 		timestamp := time.Unix(int64(jf.Fields["timestamp"].(float64)), 0)
-		jf.Fields["executor_id"] = task.ExecutorID
-
-		acc.AddFields("mesos_tasks", jf.Fields, tags, timestamp)
+		acc.AddFields("mesos_monitor_statics", jf.Fields, tags, timestamp)
```

Comment:
What's your reasoning for changing the measurement name from `mesos_tasks` to `mesos_monitor_statistics`?